waku/v2/protocol/waku_filter/waku_filter.nim

```diff
@@ -188,6 +195,7 @@ proc init*(T: type WakuFilter, switch: Switch, rng: ref BrHmacDrbgContext, handl
 # @TODO THIS SHOULD PROBABLY BE AN ADD FUNCTION AND APPEND THE PEER TO AN ARRAY
 proc setPeer*(wf: WakuFilter, peer: PeerInfo) =
   wf.peers.add(FilterPeer(peerInfo: peer))
+  waku_filter_peers.inc()
```

Comment:
What happens if a node disconnects? Does it get decremented somewhere?

Reply:
Good point. Currently this gauge just tracks the number of filter `peers` in the `seq` and, since we don't remove any disconnected peers (yet), the metric is never decremented. It does make sense, however, to at least reset this gauge to `0` when the `wakunode` is initialised. Will add.